frontend/src/search-results/TrainingResultCard.tsx

```diff
@@ -134,7 +134,7 @@ export const TrainingResultCard = (props: Props): ReactElement => {
       </div>
       <div className="row">
         <div className="col-md-12">
-          {isTabletAndUp && <p>{boldHighlightedSection(props.trainingResult.highlight)}</p>}
+          {isTabletAndUp && <p className="result-highlight">{boldHighlightedSection(props.trainingResult.highlight)}</p>}
```

Comment:
It doesn't look like this CSS class exists for styling and would only be for test targeting.

Thoughts on using a `data-testid` attribute instead?

See https://docs.cypress.io/guides/references/best-practices#Selecting-Elements and https://depo-platform-documentation.scrollhelp.site/developer-docs/best-practices-for-using-cypress#BestpracticesforusingCypress-data-testidAttribute

Reply:
Updated to use `data-testid`.